renderapi/pointmatch.py

```diff
@@ -30,6 +43,20 @@ def get_matchcollection_owners(host=None, port=None,
 @renderaccess
 def get_matchcollections(owner=None, host=None, port=None,
                          session=requests.session(), render=None, **kwargs):
+    '''get all the matchCollections owned by owner
+
+    :func:`renderapi.render.renderaccess` decorated function
+
+    Args:  
+        owner (str): matchCollection owner (fallback to render.DEFAULT_OWNER)
+                     (note match owner != stack owner always) 
+        render (RenderClient): RenderClient connection object
+        session (requests.session.Session): requests session
+    Returns:
+        list[str]: matchcollections owned by owner

```

Comment:
Maybe it's best to be explicit that these returns are unicode, we just tend to treat them as strings downstream.